docs/docs/configuration/providers/github.md

```diff
@@ -14,30 +14,42 @@ org and team or collaborators.
 
 To restrict by organization only, include the following flag:
 
-    -github-org="": restrict logins to members of this organisation
+```
+    --github-org="": restrict logins to members of this organisation
+```
 
 To restrict within an organization to specific teams, include the following flag in addition to `-github-org`:
 
-    -github-team="": restrict logins to members of any of these teams (slug), separated by a comma
+```
+    --github-team="": restrict logins to members of any of these teams (slug), separated by a comma
+```
 
 If you would rather restrict access to collaborators of a repository, those users must either have push access to a 
 public repository or any access to a private repository:
 
-    -github-repo="": restrict logins to collaborators of this repository formatted as orgname/repo
+```
+    --github-repo="": restrict logins to collaborators of this repository formatted as orgname/repo
```

Comment:
```suggestion
```shell
    --github-repo=""  # restrict logins to collaborators of this repository formatted as orgname/repo